test/integ_tests/mock_utils.py

```diff
@@ -165,28 +171,48 @@ def __init__(self):
         braket.aws.aws_session.AwsSession.create_job = AwsSessionFacade.create_job
         braket.aws.aws_session.AwsSession.get_job = AwsSessionFacade.get_job
         braket.aws.aws_session.AwsSession.cancel_job = AwsSessionFacade.cancel_job
+        braket.aws.aws_session.AwsSession.copy_s3_directory = AwsSessionFacade.copy_s3_directory
         md.CwlInsightsMetricsFetcher._get_metrics_results_sync = AwsSessionFacade.get_job_metrics
         braket.aws.aws_quantum_job.AwsQuantumJob._attempt_results_download = mock.Mock()
-        AwsSessionMinWrapper.parse_mock_devices()
+        AwsSessionMinWrapper.parse_device_config()
 
     @staticmethod
-    def parse_mock_devices():
+    def parse_device_config():
         mock_device_config_str = os.getenv("MOCK_DEVICE_CONFIG")
         if mock_device_config_str:
             AwsSessionFacade.mock_device_config = json.loads(mock_device_config_str)
         else:
             AwsSessionFacade.mock_device_config = {}
+        unsupported_device_config_str = os.getenv("UNSUPPORTED_DEVICE_CONFIG")
+        if unsupported_device_config_str:
+            AwsSessionFacade.unsupported_device_config = set(json.loads(unsupported_device_config_str))
+        else:
+            AwsSessionFacade.unsupported_device_config = {}
```

Comment:
nit: Would this be cleaner? 
```suggestion
            AwsSessionFacade.unsupported_device_config = set(json.loads(unsupported_device_config_str)) if unsupported_device_config_str else {}
```

Reply:
Sure, it's more pythonic. I'll also change the if statement above, similarly.